packages/core/types/src/common/config-module.ts

```diff
@@ -33,6 +33,10 @@ export type AdminOptions = {
    * The default value is `./build`.
    */
   outDir?: string
+  /**
+   * The URL of your Medusa server. Defaults to an empty string, which means requests will hit the same server that serves the dashboard.
+   */
+  backend?: string
```

Comment:
should we call it `backendUrl` instead? I am fine either way, but it doesn't hurt to be explicit.